meson.build

```diff
@@ -8,9 +8,9 @@
 project(
   'catch2',
   'cpp',
-  version : '3.1.0', # CML version placeholder, don't delete
+  version: '3.1.0', # CML version placeholder, don't delete
```

Comment:
This would have to be updated at https://github.com/catchorg/Catch2/blob/4db8b50aab9f91f31b2953455091e257942ec188/tools/scripts/releaseCommon.py#L96-L97

Reply:
dunno, this is just a formatting PR.

Reply:
I presume the formatting PR would prefer to not have this formatting be constantly overwritten and reverted to the previous formatting by a version bumping script...

Reply:
yeah I see what you mean. Updated.